app/src/main/java/com/example/drawingApp/MainViewModel.kt

```diff
@@ -9,5 +9,6 @@ class MainViewModel : ViewModel() {
 
     @ColorInt
     var primaryColor = Color.BLACK
+    val layers: MutableList<Layer> = mutableListOf()
```

Comment:
Not yet

Reply:
removed the data class list, for now it is a string list for the purpose of the dialog